packages/block-library/src/query-pagination/block.json

```diff
@@ -23,7 +23,15 @@
 		"html": false,
 		"color": {
 			"gradients": true,
-			"link": true
+			"link": true,
+			"__experimentalDefaultControls": {
```

Comment:
Most of the children of this block are rendered as links. Should we add `link` as one of the defaults?